### ISSUE TITLE
Scarlet Medic - (10605)

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#907]
> Constantly buffing Arcane resistance on themselves and just hitting with their staff. No option to cast arcane resistance on the player when Mindcontrolled like it was in classic
`.go xyz 1916.230000 -1338.950000 62.799300 0 2.731070`

> May be spawned out of place, a group of Scarlets are in Felstone Field walking among the Scourge.
`.go xyz 1764.500000 -1282.310000 62.180900 0 0.439077`

> Constantly spams the Arcane Resist buff while in combat.
`.go xyz 1763.180000 -1282.930000 62.411200 0 0.439077`

> They are supposed to be hostile with the scourge but they do not fight even though they are a few yards from each other
`.go xyz 1869.410000 -1643.660000 61.604900 0 0.729818`

> I cannot purge this enemies buffs off. I activate the spell but nothing comes off
`.go xyz 1843.500000 -1357.050000 61.594600 0 0.835064`

> These SPAM resist arcane on themselves over and over.
`.go xyz 1896.470000 -1342.850000 59.361500 0 2.248640`

> these spam cast Resist Arcane on themselves
`.go xyz 2015.880000 -1783.590000 62.795300 0 4.212290`

> They are not fighting the undead
`.go xyz 1789.470000 -1207.580000 59.402300 0 4.985870`

> when fighting with this mob you get a clearcast type noise every 2 seconds or so, along with the clearcasting animation on the mob.

steps to reproduce: fight the mob and listen :D
`.go xyz 2030.140000 -1778.690000 61.922300 0 0.117998`

> constantly applies its resist arcane ability when in combat, seems to tick every second
`.go xyz 1843.000000 -1208.550000 64.096300 0 1.405210`

> Does not attack nearby undead.
`.go xyz 1875.780000 -1622.190000 60.488600 0 5.993990`

> just chillin not attacking undead in the plaguelands at felstone field
`.go xyz 1753.470000 -1290.930000 63.891700 0 1.038170`

> procing something on every hit, sounds like clearcasting
`.go xyz 1884.510000 -1369.530000 64.119300 0 1.706750`

**Comment by Shadownflame** [comment]
Related https://github.com/Project-Epoch/bugtracker/issues/8070

**Comment by Bennylavaa** [comment]
this is not fixed not sure why its marked as testing

**Comment by Shadownflame** [comment]
There was a fix for npcs spamming buffs a few updates ago. Seems this npc wasn't affected.

**Comment by kaytotes** [comment]
Yeah this was a possible candidate for the other fix doing something about it. Just another edge case will take a concentrated look at this one which will be proxy fix others I'm sure.
